git-imerge

```diff
@@ -2863,8 +2863,9 @@ class MergeState(Block):
                 self.git.get_log_message(orig).rstrip('\n')
                 + '\n\n(rebased-with-history from commit %s)\n' % orig
                 )
-            commit = self.git.commit_tree(tree, [commit, orig], msg=msg)
-
+            kwargs = dict(tree=tree, parents=[commit, orig], msg=msg)
+            kwargs['metadata']=self.git.get_author_info(orig)
+            commit = self.git.commit_tree(**kwargs)
```

Comment:
That's an improvement on #119, but now that this behavior is always in effect, there is no need to use `kwargs`:

    commit = self.git.commit_tree(tree, [commit, orig], msg=msg, metadata=self.git.get_author_info(orig))

I've implemented that, and also the analogous change for the `border` variants, in #138. Let me know what you think.

Reply:
Good point!